app/controllers/mixins/ems_common_angular.rb

```diff
@@ -392,6 +424,13 @@ def build_credentials(ems, mode)
                          :save          => (mode != :validate)}
         session[:oauth_response] = nil
       end
+      if ems.kind_of?(ManageIQ::Providers::ContainerManager) &&
+         ems.supports_authentication?(:bearer) && params[:bearer_password]
+        creds[:hawkular] = {:auth_key => params[:bearer_password], :userid => "_"}
+        creds[:bearer] = {:auth_key => params[:bearer_password]}
+      end
+
+      ems.update_authentication(creds, :save => (mode != :validate))
```

Comment:
`ems.update_authentication(creds, :save => (mode != :validate))`

@AparnaKarve  without this line the .`authentication_token("bearer")` is not set :-(
1. is their a better way to do this ?
2. do we need it ?
3. do we need it only for container-manager ?